jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -494,7 +236,7 @@ It is possible to run your config.yml against your organization's policies outsi
 The following command will request a decision for the provided config input and return a Circle Decision containing the status of the decision
 and any violations that may have occurred. 
 
-# TODO: add links to circleci-cli/references
+#TODO: add links to circleci-cli/references#
```

Comment:
@sagar-connect could you check if this "To do" has been addressed?

Reply:
I'm not even sure what we would be linking to in circleci-cli can @sagar-connect or @davidmdm let us know if we need this still? If we don't hear back in a day let's just remove the TODO

Reply:
I think this point is suggesting to include links to `decision` command in CLI.
https://github.com/circleci/circleci-docs/blob/DOCTEAM-765-config-policy/jekyll/_cci2/use-the-cli-for-config-and-policy-development.adoc#developing-policies